frontend/src/components/StationMiniCard/StationMiniCard.scss

```diff
@@ -0,0 +1,52 @@
+@import "../../variable.scss";
+
+.station_mc {
+  box-sizing: border-box;
+  height: 5rem;
```

Comment:
Change the unit of measurement

Reply:
this value is fixed on purpose, rem is the good unit here.